java/src/shared/facades/TurnFacade.java

```diff
@@ -168,7 +168,7 @@ public void calcVictoryPoints() {
         for (Player p : getModel().getPlayers()) {
             int points = p.getMonuments();
             points += getModel().getMap().getPlayerSettlements(p.getPlayerIndex()).size();
-            points += getModel().getMap().getPlayerCities(p.getPlayerIndex()).size();
+            points += (getModel().getMap().getPlayerCities(p.getPlayerIndex()).size() * 2);
```

Comment:
Derp - that's what I get for coding this at the last minute and freaking out. Don't really need the parentheses.